persistent-postgresql/Database/Persist/Postgresql.hs

```diff
@@ -1029,6 +1045,7 @@ getColumn getter tableName' [ PersistText columnName
                 , "WHERE tc.constraint_type='FOREIGN KEY' "
                 , "AND kcu.ordinal_position=1 "
                 , "AND kcu.table_name=? "
+                , "AND kcu.table_schema=? "
                 , "AND kcu.column_name=? "
                 , "AND tc.constraint_name=?"
                 ]
```

Comment:
This `information_schema` query change is a non-trivial change to the way Persistent and Postgres integrate. I think that my change is in the spirit of the original, but reviewers should verify for themselves that this makes sense.